src/components/Notification.tsx

```diff
@@ -0,0 +1,80 @@
+import classnames from 'classnames';
+import { HeadingLevel } from '../types/headingLevel';
+import { Icon } from './Icon';
+import './Notification.css';
```

Comment:
FWIW vite can handle Less stylesheets so you can change `Notification.css` to `Notification.less` to use Less syntax if preferred.